src/test/java/RoundTripSearchTest.java

```diff
@@ -0,0 +1,48 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+import pages.MainPage;
+import pages.FlightSearchPage;
+
+import static org.testng.Assert.assertTrue;
+
+public class RoundTripSearchTest {
+    private WebDriver driver;
+    private MainPage mainPage;
+    private FlightSearchPage flightSearchPage;
+
+    @BeforeClass
```

Comment:
How often should we setup driver and open the app? Is beforeClass engough?
How many tests can be in the class? Will environment be clean in this case?